src/core_plugins/getting_started/public/components/getting_started/getting_started.html

```diff
@@ -149,20 +132,20 @@ <h2 class="kuiSubTitle">
     </div>
   </div>
 
-  <div class="kuiViewContentItem kuiVerticalRhythm">
+  <div
+    ng-if="!gettingStarted.hasOptedOut()"
+    class="kuiViewContentItem kuiVerticalRhythm"
+  >
     <div class="kuiFieldGroup">
       <div class="kuiFieldGroupSection">
-        <p class="kuiText">Just want to see what Kibana is capable of doing?</p>
-      </div>
-
-      <div class="kuiFieldGroupSection">
         <a
           class="kuiButton kuiButton--primary"
-          ng-href="{{ gettingStarted.documentationLinks.demoSite }}"
-          target="_blank"
+          kbn-href="#/management"
+          kbn-getting-started-opt-out
+          data-test-subj="lnkGettingStartedOptOut"
         >
           <span class="kuiButton__inner">
-            <span>View the demo site</span>
+            <span>Got it, take me to Kibana</span>
```

Comment:
Yes, they should be! Rather than increase the scope of this PR, I'll do the localization in a new PR. I've created https://github.com/elastic/kibana/issues/12004 to track it. Thanks!